CHANGELOG.md

```diff
@@ -38,6 +40,33 @@ If we missed any change or there's something you'd like to discuss about this ve
     + `Spotify::prompt_for_user_token[_without_cache]`
     + The `ClientError::CLI` variant, for whenever user interaction goes wrong
 - Fix typo in `user_playlist_remove_specific_occurrenes_of_tracks`, now it's `user_playlist_remove_specific_occurrences_of_tracks`.
+- All fallible calls in the client return a `ClientError` rather than using `failure`.
+- Endpoints take `Vec<String>/&[String]` as parameter have changed to `impl IntoIterator<Item = &str>`, which is backward compatibility.
+  + The endpoints which changes parameter from `Vec<String>` to `impl IntoIterator<Item = &str>`:
+	- `artists`
+	- `albums`
+	- `save_shows`
+	- `get_several_episodes`
+	- `check_users_saved_shows`
+	- `remove_users_saved_shows`
+  + The endpoints which changes parameter from `&[String]` to `impl IntoIterator<Item = &str>`:
+	- `user_playlist_add_tracks`
+	- `user_playlist_replace_tracks`
+	- `user_playlist_remove_all_occurrences_of_tracks`
+	- `current_user_saved_tracks_delete`
+	- `current_user_saved_tracks_contains`
+	- `current_user_saved_tracks_add`
+	- `current_user_saved_albums_add`
+	- `current_user_saved_albums_delete`
+	- `current_user_saved_albums_contains`
+	- `user_follow_artists`
+	- `user_unfollow_artists`
+	- `user_artist_check_follow`
+	- `user_follow_users`
+	- `user_unfollow_users`
+	- `audios_features`
+- Reexport `model` module to allow user to write `rspotify::model::FullAlbum` instead of  `rspotify::model::FullAlbum`.
```

Comment:
I think you got confused here

Reply:
Sorry for not getting your point.

Reply:
I just don't understand this line. Both import paths are the same.

Reply:
Ooops, I get it now. I'll fix it.